### DISCUSSION TITLE
Close some old issues

### DISCUSSION THREAD

**Discussion by mths1** [vernemq/vernemq#2264] [General]
There are quite some issues that are already resolved, but still open. Among maybe others, those some to be fixed:

760
1400
1412
1414 
1462
1514
1605
1634
1671
1673
1676
1708
1754
1828
1897
2097

**Comment by ioolkos** [comment]
@mths1 +1, thanks for the suggestion. Will start going through those. Some are "support" requests but I think we can close a lot of those as well.

**Reply by ioolkos** [reply]
All of the above are closed now. Let me know if you have more, @mths1, and thanks for resolving a lot of those with your contribs!
:1st_place_medal:

**Reply by mths1** [reply]
Some more that look like fixed alread:
#1017
#1127
#1203
#1444
#1781
#1908
